plugins/chat/assets/stylesheets/mobile/chat-message-actions.scss

```diff
@@ -154,15 +154,15 @@
   height: 100%;
   width: 100%;
   z-index: z("header") + 1;
-  transition: background-color 0.4s ease;
+  transition: background-color 0.4s ease-in;
 
   .collapse-area {
     width: 100%;
     height: 100%;
   }
 
   &.fade-in {
-    background: rgba(var(--always-black-rgb), 0.75);
+    background: rgba(var(--always-black-rgb), 0.6);
```

Comment:
Cool! 

Sidenote: I often find selecting something, then immediately want to refer to it (for example to read it more closely, to remember which one I selected). A lighter backdrop is probably going to help!